starfish/munge.py

```diff
@@ -37,6 +37,32 @@ def gather(df, key, value, cols):
     return pd.melt(df, id_vars, id_values, var_name, value_name)
 
 
+def spots_to_geojson(r):
```

Comment:
this function already lives in viz.py::spots_geo_json. we have three options:

1. delete existing function in viz and use the one in munge. merge PR as is.
2. move your function to viz to replace it. fix imports.
3. use function in viz. remove changes to munge. update cli with new paths and functions

i'm in favor of 1 because this really is more of a munge operation than a viz operation and sees simplest. what do you think?